spec/registration/create.account.successful.spec.js

```diff
@@ -0,0 +1,18 @@
+const mainPage = require("../../page_objects/main.page");
+const loginPage = require("../../page_objects/login.page");
+const registrationPage = require("../../page_objects/registration.page");
+
+describe("Hotline registration", () => {
+    beforeAll(async function () {
+       await browser.restart();
+       browser.waitForAngularEnabled(false);
```

Comment:
`await browser.restart();
browser.waitForAngularEnabled(false);`
no need to do it here, it is already done in config->onPrepare

Reply:
removed beforeAll from tc and updated conf.js